JSDemos/Demos/Scheduler/AllDayPanelMode/description.md

```diff
@@ -0,0 +1,10 @@
+The **All day** Scheduler panel displays appointments that last all day or have the [allDay](/Documentation/ApiReference/Common/Object_Structures/dxSchedulerAppointment/#allDay) property set to **true**. To change the **All day** panel display mode, assign one of the following values to the [allDayPanelMode](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/#allDayPanelMode) property:
```

Comment:
Did you mean to say something like this in the first sentence? 

```suggestion
The **All day** Scheduler panel displays appointments that last over 24 hours or have the [allDay](/Documentation/ApiReference/Common/Object_Structures/dxSchedulerAppointment/#allDay) property set to **true** (don't have specific start/end times). To change the **All day** panel display mode, assign one of the following values to the [allDayPanelMode](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/#allDayPanelMode) property:
```

Or maybe this: 

```suggestion
The **All day** Scheduler panel displays appointments that last all day (have the [allDay](/Documentation/ApiReference/Common/Object_Structures/dxSchedulerAppointment/#allDay) property set to **true**). To change the **All day** panel display mode, assign one of the following values to the [allDayPanelMode](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/#allDayPanelMode) property:
```